src/picongpu/include/particles/ParticlesFunctors.hpp

```diff
@@ -155,29 +155,66 @@ struct CallUpdate
     }
 };
 
-/* Tests if species can be ionized and calls the function to do that */
+/** \struct CallIonization
```

Comment:
yes, you only need those if not put directly over the classes (which would be messy).

the only thing one needs of those is actually `\file` for a general file description.